docs/semgrep-supply-chain/getting-started.md

```diff
@@ -68,55 +58,34 @@ Dependency scans can be triggered by the following, depending on your CI set up:
   <tr>
    <td>Pull or merge request
    </td>
-   <td>diff-aware scan
+   <td><a href="/semgrep-ci/running-semgrep-ci-with-semgrep-cloud-platform/#diff-aware-scanning">Diff-aware scan</a>
    </td>
    <td>All dependency rules
    </td>
   </tr>
   <tr>
    <td>Push or scheduled event, such as a cron job
    </td>
-   <td>full scan
+   <td>Full scan
    </td>
    <td>All dependency rules
    </td>
   </tr>
 </table>
 
-For more information on diff-aware and full scans, see [Diff-aware scanning](/docs/semgrep-ci/running-semgrep-ci-with-semgrep-cloud-platform/#diff-aware-scanning).
-
-### Blocking a PR or MR
-
-Both reachable and unreachable findings of Semgrep Supply Chain do **not** block a pull request or merge request.
-
-:::info
-Semgrep versions v0.122.0 and below previously blocked reachable findings.
-:::
-
-### Setting a daily scan schedule
-
-Semgrep Supply Chain frequently releases new rules. By default, Semgrep Supply Chain scans your codebase once a day. Change your time or frequency through the following references:
-
-<CiScheduling />
-
-## Ignoring dependency findings through `semgrepignore`
+## Scan a monorepo's dependencies
 
-See [Ignoring dependency findings](/docs/semgrep-supply-chain/ignoring-lockfiles-dependencies).
+Semgrep Supply Chain supports the scanning of monorepos. It treates each of the monorepo's subdirectories as an individual repository. Findings are then grouped by the repositories based on the [lockfile](/semgrep-supply-chain/glossary/#lockfile) or manifest file present in the monorepo.
 
-## Triaging and remediating dependency findings
 
-Semgrep Supply Chain enables developers to perform triage and remediation through the **[Vulnerabilities](https://semgrep.dev/orgs/-/supply-chain/vulnerabilities)** page. On this page you can perform the following actions:
+## Block pull/merge requests
 
-* View specific **reachable** vulnerable lines of code in your codebase. This helps to evaluate the threat.
-* View specific lines of code where your dependency is being declared.
-* Triage a dependency finding.
-* Attach a PR or MR, or Jira ticket to the finding.
-* Upgrade the dependency that generated the finding to a safe version. A safe version is any newer version of the dependency that does not contain the vulnerability. This resolves the finding.
+Though Semgrep Supply Chain v0.122.0 and earlier automatically blocked pull/merge requests if it discovered reachable findings in the code, it no longer does this. You can, however, configure your version control system to prevent merging if Semgrep Supply Chain identifies reachable findings.
```

Comment:
```suggestion
Though Semgrep Supply Chain **v0.122.0** and earlier automatically blocked pull/merge requests if it discovered reachable findings in the code, it no longer does this. You can, however, configure your version control system to prevent merging if Semgrep Supply Chain identifies reachable findings.
```